ciao-deploy/deploy/cnci.go

```diff
@@ -81,6 +82,27 @@ func unMountImage(ctx context.Context, devPath string, mntDir string) error {
 	return errOut
 }
 
+func getProxy(upper, lower string) (string, error) {
```

Comment:
I don't think it's worth taking these strings as parameters. Just hardcode them in this function.